osu.Game/OsuGame.cs

```diff
@@ -817,7 +817,8 @@ protected override void LoadComplete()
                 },
                 topMostOverlayContent = new Container { RelativeSizeAxes = Axes.Both },
                 idleTracker,
-                new ConfineMouseTracker()
+                new ConfineMouseTracker(),
+                new InvalidRulesetKeyBindingHandler { PostNotification = n => Notifications.Post(n) }
```

Comment:
This is *super* weird. I really dislike having this component at a top level like this.